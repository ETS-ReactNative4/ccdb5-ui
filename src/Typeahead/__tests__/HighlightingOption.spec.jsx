import React from 'react'
import HighlightingOption from '../HighlightingOption'
import renderer from 'react-test-renderer'

describe('component::HighlightingOption', () => {
  describe('initial state', () => {
    it('renders without crashing', () => {
      const target = renderer.create(
        <HighlightingOption label="Foobar" position={0} value="foo" />
      )

      const tree = target.toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})

