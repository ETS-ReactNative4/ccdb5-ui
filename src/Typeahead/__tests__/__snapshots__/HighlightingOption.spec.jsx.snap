// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::HighlightingOption initial state renders without crashing 1`] = `
<span>
  
  <b>
    Foo
  </b>
  bar
</span>
`;
